modules/powers/Nemesis.class.php

```diff
@@ -15,7 +15,148 @@ public function __construct($game, $playerId)
     $this->playerCount = [2, 3, 4];
     $this->golden  = false;
     $this->orderAid = 42;
+
+    $this->implemented = true;
+  }
+
+
+  public function stateAfterBuild()
+  {
+    $arg = [];
+    $this->argUsePower($arg);
+    return (count($arg['workers']) > 0)? 'power' : null;
+  }
+
+
+  public function argUsePower(&$arg)
+  {
+    $arg['power'] = $this->id;
+    $arg['power_name'] = $this->name;
+    $arg['skippable'] = true;
+    $arg['workers'] = $this->game->board->getPlacedActiveWorkers();
+    foreach($arg['workers'] as &$worker)
+    {
+      $worker['works'] = [];
+    }
+	$workers = $this->game->board->getPlacedActiveWorkers();
+	if (count($workers) != 2)
```

Comment:
The original (box) power allowed to play Nemesis vs killer powers, but not the updated power which clearly says that Nemesis must have two workers and opponents must have at least two workers

Reply:
Actually, I'm now wondering what happens in Nemesis vs 1-worker Hydra. The updated text does not seem to handle this case. I'll ask it on discord.

Reply:
In the last commit I changed the power to ignore a single-head hydra instead of raising an error, as the updated description suggests. Might also ban hydra to avoid this issue.